Samples/excel-shared-runtime-scenario/README.md

```diff
@@ -64,17 +61,20 @@ The add-in is aware of whether it is connected. When connected you will see the
 
 Additionally the add-in has a custom function that can display a filtered view of the data. The custom function is aware of the connection status, so that when connected, it will display the mock data. When disconnected, it will show `#N/A`.
 
-## Create certificate files
-
-The addin needs HTTPS website. Please run `npx office-addin-dev-certs install --days 365` to install dev certificates.
-
-## Build and run the solution
+## Run the sample
 
-In the command prompt in the root of the project, run the command `npm install`.
+You can run this sample in Excel in a browser. The add-in web files are served from this repo on GitHub.
 
-When the installation completes, run the command `start npm start`. This will open a second command prompt, build the project and then start a server (with dev mode settings). It takes from 5 to 30 seconds. When it finishes, the last line should say `Compiled successfully`. Minimize this command prompt.
-
-Back in the original command prompt, run the command `npm run sideload`. This will launch Excel and sideload the add-in. After a few seconds, a ribbon named `Contoso Data` will appear.
+1. Download the **manifest.xml** file from this sample to a folder on your computer.
+1. Open [Office on the web](https://office.live.com/).
+1. Choose **Excel**, and then open a new document.
+1. Open the **Insert** tab on the ribbon and choose **Office Add-ins**.
+1. On the **Office Add-ins** dialog, select the **MY ADD-INS** tab, choose **Manage My Add-ins**, and then **Upload My Add-in**.
+   ![The Office Add-ins dialog with a drop-down in the upper right reading "Manage my add-ins" and a drop-down below it with the option "Upload My Add-in"](../../images/office-add-ins-my-account.png)
+1. Browse to the add-in manifest file, and then select **Upload**.
```

Comment:
There technically two manifest files here: the standard and one for localhost. Is that intentional?

Reply:
Yes, later in the readme we explain how to set up your own localhost if you prefer.